src/Sentry/Extensibility/ExceptionExtensions.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Collections.Generic;
+using Sentry.Internal;
+
+namespace Sentry.Extensibility
```

Comment:
Users will need to import this namespace to see the extension methods.
Better on `Sentry` namespace. `System` I wonder could be even better since we're naming the method all `XyzSentryQwe`

Reply:
the root folder will end up with a lot of source files D:
but I agree the namespace should only be "Sentry"

Reply:
Please don't put it on `System`, it's not good user experience. As a user, I don't want to see random extension methods appearing on `Exception` after I import `System` in my code.

Also, if you want to make a global extension method, using global namespace is better than assuming the user has `System` imported.

Reply:
I thought about thr global namespace, agree it's better than System which has no benefits. It's not bad use rexperience to show the error tracking api on an exception instance. We're not polluting String or Int with random helpers.

Reply:
If you have a custom Exception, there's a risk the source file that caught the custom exception might not be using the System namespace, so it might be strange to the end-user that it requires to include the System namespace to use a Sentry extension instead of Sentry namespace.

Reply:
> I thought about thr global namespace, agree it's better than System which has no benefits. It's not bad use rexperience to show the error tracking api on an exception instance. We're not polluting String or Int with random helpers.

I don't know, for me personally that's not a good UX. I would prefer to import `Sentry` or even `Sentry.Extensions` rather than have extension methods forced on me without any way to opt out. Then, when I will want to find out where that method comes from, I will see that it's defined in `System` namespace and that will be extremely confusing. Or worse, like @lucas-zimerman said, when I already have an instance of `Exception` and then have to import `System` to use Sentry-specific methods, also confusing.

Reply:
`System` was a bad idea.

Global namespace is an excellent idea. The problem is discoverability. 

You'd need to known that you must `using Sentry` to see that. And Sentry is what you use to better instrument your app for errors, extending `Exception` is a great candidate.

I've seen extension methods such as `exception.Capture()` which I'm not a fan at all. But extension such as what @lucas-zimerman added sounds great.